.aspect/workflows/terraform/workflows.tf

```diff
@@ -0,0 +1,102 @@
+provider "aws" {
+  alias = "workflows"
+
+  region = "us-west-2"
+
+  default_tags {
+    tags = {
+      (module.aspect_workflows.cost_allocation_tag) = module.aspect_workflows.cost_allocation_tag_value
+    }
+  }
+}
+
+data "aws_ami" "runner_ami" {
+  # Aspect's AWS account 213396452403 provides public Aspect Workflows images for getting started
+  # during the trial period. We recommend that all Workflows users build their own AMIs and keep
+  # up-to date with patches. See https://docs.aspect.build/v/workflows/install/packer for more info
+  # and/or https://github.com/aspect-build/workflows-images for example packer scripts and BUILD
+  # targets for building AMIs for Workflows.
+  owners      = ["213396452403"]
+  most_recent = true
+  filter {
+    name   = "name"
+    values = ["aspect-workflows-buildkite-al2-gcc-*"]
+  }
+}
+
+module "aspect_workflows" {
+  providers = {
+    aws = aws.workflows
+  }
+
+  # Aspect Workflows terraform module
+  source = "https://s3.us-east-2.amazonaws.com/static.aspect.build/aspect/5.7.0-rc5/workflows/terraform-aws-aspect-workflows.zip"
+
+  # Non-terraform Aspect Workflows release artifacts are pulled from the region specific
+  # aspect-artifacts bucket during apply. Aspect will grant your AWS account access to this bucket
+  # during the trial setup. The aspect-artifacts bucket used must in the same region as the
+  # deployment.
+  aspect_artifacts_bucket = "aspect-artifacts-us-west-2"
+
+  # Name of the deployment
+  customer_id = "aspect-build/rules_jest"
+
+  # VPC properties
+  vpc_id             = module.vpc.vpc_id
+  vpc_subnets        = module.vpc.private_subnets
+  vpc_subnets_public = []
+
+  # Whether or not to allow SSM access to runners
+  enable_ssm_access  = true
+
+  # Opt-in to k8s HA remote cache.
```

Comment:
HA?

Reply:
highi-availability . should probably expand the acronym for customers not famiilar with it